app/components/Documentation/components/Pagination/PaginationDocumentation.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+import { getIpfsHashFromBytes32 } from 'utils/ipfs';
+import createdHistory from 'createdHistory';
+import * as routes from 'routes-config';
+import useMediaQuery from 'hooks/useMediaQuery';
+import { getDataFromTree, getcurrentArrayIndex } from '../../helpers';
+import { DocumentationItemMenuType } from 'pages/Documentation/types';
+import ButtonPaginationDesktop from './ButtonPaginationDesktop';
+import ButtonPaginationMobile from './ButtonPaginationMobile';
+import { PREV_TYPE_BUTTON, NEXT_TYPE_BUTTON } from './constants';
+
+type PaginationDocumentationProps = {
+  documentationMenu: DocumentationItemMenuType;
+  id: number;
+  onClickPaginationArticleEditDocumentation?: (arrayId: string) => void;
+};
+
+const PaginationDocumentation: React.FC<PaginationDocumentationProps> = ({
+  documentationMenu,
+  id,
+  onClickPaginationArticleEditDocumentation,
+}): JSX.Element => {
+  const isDesktop = useMediaQuery('(min-width: 992px)');
+  const treeArray = getDataFromTree(documentationMenu);
+
+  const currentArrayIndex = getcurrentArrayIndex(treeArray, id);
+  const isStartArticle = currentArrayIndex < 1;
+  const isLastArticle = currentArrayIndex === treeArray.length - 1;
+
+  const getButtonContent = (
+    typeButton: typeof PREV_TYPE_BUTTON | typeof NEXT_TYPE_BUTTON,
+  ) => {
+    const stepDirection = typeButton === PREV_TYPE_BUTTON ? -1 : +1;
+    return treeArray.find(
+      (item) => item?.treeIndex === currentArrayIndex + stepDirection,
+    )?.node;
+  };
+
+  const onClickPaginationArticle =
+    (typeButton: typeof PREV_TYPE_BUTTON | typeof NEXT_TYPE_BUTTON) => () => {
+      const { id, title } = getButtonContent(typeButton);
+
+      if (typeof onClickPaginationArticleEditDocumentation === 'function') {
```

Comment:
Same check as above, do you need it?

Reply:
It was Denis Romanov's decision. We can fix this during refactoring.

Reply:
Confirm